lib/gssapi/mech/gss_inquire_cred.c

```diff
@@ -99,7 +99,8 @@ gss_inquire_cred(OM_uint32 *minor_status,
 			gss_name_t mc_name = GSS_C_NO_NAME;
 			OM_uint32 mc_lifetime = GSS_C_INDEFINITE;
 
-			if (mc->gmc_mech->gm_inquire_cred == NULL)
+			if (mc->gmc_mech->gm_inquire_cred == NULL ||
+			    (mc->gmc_mech->gm_flags & GM_USE_MG_CRED))
```

Comment:
But now we'll never have a mech cred element for a `GM_USE_MG_CRED` mech, no?

Reply:
Well, never say never. Perhaps it wants to customize the output but still use mechglue credentials. Or, if you think it's a hard guarantee we should change this to an assert.

Reply:
Hmm, ok.